docs/hyperloop/notifications.md

```diff
@@ -0,0 +1,105 @@
+---
+sort: 3
+title: Notifications
+---
+
+## <a name="notifications"></a>Notifications
+* There are different types of notifications that the user receives, each related to a specific topic. A notification does not necessarly request an action to be taken, but informs the user aboout a specific element, and depending on the situation, the user can decide the possible next steps.
+
+### <a name="datasetNotifications"></a>Dataset notifications
+
+#### <a name="datasetChanged"></a>Dataset changed
+
+* Notifies the user that an update has been made to one of the datasets included in his analyses. The notification message contains a link to the history of the dataset that has been modified, as well as a link to the analysis this dataset is included in. The update here indicates that:
+  * The name or discription of the dataset has changed
+  * The dataset has been staged or unstaged to a target
+  * A linked dataset had been added or removed
+  * The automatic composition schedule has been changed
+
+<div align="center">
+    <img src="../images/datasetChanged.png" width="90%">
+</div>
+
+#### <a name="datasetActivated"></a>Dataset activated / deactivated
+
+* Notifies the user when a dataset has been successfully activated or deactivated.
+
+<div align="center">
+    <img src="../images/datasetActivation.png" width="90%">
+</div>
+
+#### <a name="productionAdded"></a>Production added 
+
+* Informs the user that a production has been successfully created for the dataset.
+
+<div align="center">
+    <img src="../images/productionAdded.png" width="90%">
+</div>
+
+
+#### <a name="trainrunAdded"></a>Train run added into / removed from dataset
```

Comment:
This is only for converted Run 2 data. 
We should say here "Conversion train run" instead of "train run" to make it more clear